Makefile

```diff
@@ -252,3 +252,6 @@ test-yaml:
 	@which yamllint >/dev/null 2>&1 || echo "Command 'yamllint' not found, can not execute YAML syntax checks"
 	@# Fall back to find if there is no git, e.g. in package builds
 	yamllint --strict $$((git ls-files "*.yml" "*.yaml" 2>/dev/null || find -name '*.y*ml') | grep -v ^dbicdh)
+
+update-deps:
+	perl etc/update-deps.pl
```

Comment:
@perlpunk your change inspired me to do https://github.com/os-autoinst/openQA/pull/2939 . Hence I suggest to provide update-deps.pl not in "etc" which should be reserved for configuration files to be installed on systems but either – after my PR – in tools or in script and then deleted on install in Makefile

Reply:
moved it to `tools/`.
Do I need to delete it explicitly or is this directory already removed on install?

Reply:
I see no delete happening in #2939/master

Reply:
Exactly, not needing an explicit mention what was I wanted to achieve